json_merger/comparator.py

```diff
@@ -29,8 +29,17 @@
 
 
 class BaseComparator(object):
+    """Abstract base class for Entity Comparison."""
 
     def __init__(self, l1, l2):
+        """
+        Do not override the constructor since it will be called by the
+        json_merger logic.
```

Comment:
:bike: :bike: :bike: Older logic was using self.l1 and self.l2. But whatever this is Python, devs should know what they do when they inherit from things. The example that I have shows you how to override two things.